docs/Getting Started/Installation/index.md

```diff
@@ -51,6 +51,19 @@ pod install
 open <#Your Project Name#>.xcworkspace
 ```
 
+### Swift Package Manager Installation
```

Comment:
The introduction header for the **Installation** section (https://smartdevicelink.com/en/guides/iOS/getting-started/installation/#install-sdl-sdk) needs to include SPM. 

> There are four different ways to install the SDL SDK in your project: Accio, CocoaPods, Carthage, or manually.